process/errors.go

```diff
@@ -457,11 +448,11 @@ var ErrInvalidPeerAccount = errors.New("invalid peer account")
 // ErrInvalidMetaHeader signals that a wrong implementation of HeaderHandler was provided
 var ErrInvalidMetaHeader = errors.New("invalid header provided, expected MetaBlock")
 
-// ErrNilShardHeaderStorage signals that nil storage was provided for shard headers
-var ErrNilShardHeaderStorage = errors.New("nil shard header storage")
+// ErrNilEndOfEpochTrigger signals that a nil end of epoch trigger was provided
```

Comment:
You can remove also ErrTxNotValid as it seems that is not used anymore

Reply:
done